ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/NoUnusedImportsRule.kt

```diff
@@ -112,5 +126,43 @@ class NoUnusedImportsRule : Rule("no-unused-imports") {
         }
     }
 
+    // Builds a list of imports. Takes care of having aliases in case it is assigned to imports
+    private fun buildNonRedundantImports(text: String) {
+        val element = text.split("import").last().trim()
+        if (element.contains(" as ")) {
+            imports.addAll(listOf(element.split(" as ").first().replace("`", "").trim(), element.split("as").last().trim()))
```

Comment:
should that second param be checking for `" as "` too?

Reply:
Pulled the `" as "`  into a variable so as to avoid misses in check